Mathlib/Data/Finsupp/Defs.lean

```diff
@@ -334,9 +331,9 @@ lemma range_mapRange (e : M → N) (he₀ : e 0 = 0) :
   · intro h
     classical
     choose f h using h
-    use onFinset g.support (Set.indicator g.support f) (by aesop)
+    use onFinset g.support (fun x ↦ if x ∈ g.support then f x else 0) (by aesop)
```

Comment:
This is a bit disappointing, but probably fine within a proof

Reply:
That was my exact reasoning

Reply:
Why can't we keep the indicator function here?

Reply:
We can, but `Algebra.Group.Indicator` imports
* [Data.Set.SymmDiff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/SymmDiff.html)
* [Algebra.Group.Pi.Lemmas](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Group/Pi/Lemmas.html)

extra compared to `Algebra.Group.Support`, and the second one is quite heavy compared to how shallow we are in the import tree.